.github/workflows/automated-tests.yml

```diff
@@ -23,8 +23,6 @@ jobs:
         with:
           go-version: "1.19" # The Go version to download (if necessary) and use.
 
-      - name: Unit, integration and difference tests
-        run: go test ./...
-
       - name: E2E tests
-        run: make test-e2e-short
+        run: go run ./tests/e2e/... --happy-path-only
```

Comment:
```suggestion
        run: make test-e2e-short
```

Reply:
oh -- actually I'd meant to add --verbose

Reply:
Please use `make test-e2e-short`. Developers can always run the tests locally with `--verbose`.

Reply:
But we can't prove outcomes using CI, so that there's no debate, in the way that you recommend.

CI is a neutral and clearly defined build environment that can be used as proof of issues.

Reply:
We want proof.